app/javascript/src/components/Dashboard/Contacts/Table/TableColumnData.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Avatar, Dropdown } from "neetoui";
+
+import AvatarProfile from "../../../../../../assets/images/AvatarProfile";
```

Comment:
Does this import path look fine to you even after going through https://www.bigbinary.com/books/learn-rubyonrails-book/setting-up-react-environment#aliases?